chain/network/src/peer/peer_actor.rs

```diff
@@ -333,8 +339,16 @@ impl PeerActor {
                 archival: self.network_state.config.archive,
             },
             partial_edge_info: spec.partial_edge_info,
+            owned_account: self.network_state.config.validator.as_ref().map(|vc| {
+                OwnedAccount {
+                    account_key: vc.signer.public_key().clone(),
+                    peer_id: self.network_state.config.node_id(),
+                    timestamp: self.clock.now_utc(),
+                }
+                .sign(vc.signer.as_ref())
+            }),
         };
-        let msg = PeerMessage::Handshake(handshake);
+        let msg = PeerMessage::Tier2Handshake(handshake);
```

Comment:
so we're sending the owned_account inside tier2 handshake too?

Reply:
yes, see the response to the comment above